ADAPT-10819 Fix: Missing tabIndex on flipcard items

ethan-lp opened the pull request:
<!-->Link the PR to the original issue</!-->
fixes [ADAPT-10817](https://learningpool.atlassian.net/browse/ADAPT-10817)

### Fix
Ensured tabIndex on item to allow it to receive the class .focus-visible. Also updated 2 instances to uppercase 'Index' as error was flagged in console.

ethan-lp committed:
ADAPT-10819 ensuring item has tabIndex and updated 2 more

cahirodoherty-learningpool commented on templates/flipcard.jsx:
```diff
@@ -19,11 +19,12 @@ export default function flipcard(props) {
               `component__item flipcard__item item-${index} ${_flipDirection}`,
             ])}
             key={index}
+            tabIndex='0'
```
Is this one needed? I have a feeling it may disrupt the a11y flow of the component

ethan-lp replied:
It is needed to allow the item to take the focus-visible class, however with or without it I am seeing the same behaviour / a11y flow using keyboard and enabling voiceover.

eleanor-heath commented on templates/flipcard.jsx:
```diff
@@ -19,11 +19,12 @@ export default function flipcard(props) {
               className='flipcard__item-face flipcard__item-front'
               role='button'
-              tabindex='0'
+              tabIndex='0'
```
I dont think the I of index needs to be capitalised, please refer to this https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

cahirodoherty-learningpool replied:
Thats the mistake I made too Eleanor. HTML-wise you are correct, but React needs the camel case version - and then it changes this to the lowercase for output

ethan-lp committed:
removed incorrect tabIndex

ethan-lp committed:
corrections

ethan-lp commented:
@cahirodoherty-learningpool @eleanor-heath I've removed the tab index I introduced but left in the capitalised ones, this does not solve the focus ring issue but that will be address in another ticket.

cahirodoherty-learningpool reviewed (approved):
👀
